src/insexpand.c

```diff
@@ -4477,6 +4477,8 @@ ins_compl_expand_multiple(char_u *str)
     char_u	*start = str;
     char_u	*curr = str;
     int		base_indent = get_indent();
+    int		paste = p_paste;
+    p_paste = TRUE;
```

Comment:
Using the side effect of the "paste" option to fix this issue, doesn't look correct to me.  Can't you check for some other condition in the indentation code?

Reply:
thanks. i have  add a new condition on do_cindent